CHANGELOG.MD

```diff
@@ -12,6 +12,7 @@
 * Fix bug where clearing the input and entering "~/" with Helm-style
   shortcuts enabled did not automatically fill in the user's home
   directory.
+* Added option for the default input value on toggle.
```

Comment:
I hate keeping them but Atom shows a link to them if you have one and the original project had one SO IM OBLIGATED GAH